torch/ao/quantization/fx/_decomposed.py

```diff
@@ -208,7 +214,7 @@ def dequantize_per_tensor_tensor_meta(input, scale, zero_point, quant_min, quant
 # TODO: remove other variants and keep this one
 quantized_decomposed_lib.define(
     "dequantize_per_tensor.tensor2(Tensor input, Tensor scale, Tensor zero_point, "
-    "Tensor quant_min, Tensor quant_max, ScalarType dtype) -> Tensor")
+    "Tensor quant_min, Tensor quant_max, ScalarType dtype, ScalarType output_dtype) -> Tensor")
 
 @impl(quantized_decomposed_lib, "dequantize_per_tensor.tensor2", "CompositeExplicitAutograd")
 def dequantize_per_tensor_tensor2(
```

Comment:
we might be able to remove this op, maybe it's better to remove first before adding this PR